src/pages/Pool-V3/hooks/useCreatePosition.ts

```
@@ -0,0 +1,357 @@
+      );
+
+      if (tokenX && +deposit !== 0) {
+        setAmountY(deposit);
+        setAmountX(Number(amount));
+      }
+    }
+  };
+
+  const calcAmount = (amount: TokenAmount, left: number, right: number, tokenAddress: string) => {
+    if (!pool) return BigInt(0);
+    if (!tokenX || !tokenY || isNaN(left) || isNaN(right)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)